src/test/java/com/thomashan/coup/game/StandardGameScenarioTest.java

```diff
@@ -0,0 +1,134 @@
+package com.thomashan.coup.game;
+
+import com.thomashan.coup.action.Action;
+import com.thomashan.coup.action.BlockAction;
+import com.thomashan.coup.action.ChallengeAction;
+import com.thomashan.coup.action.MainAction;
+import com.thomashan.coup.action.RevealCardAction;
+import com.thomashan.coup.player.Player;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static com.thomashan.coup.action.BlockActionType.BLOCK_ASSASSINATE;
+import static com.thomashan.coup.action.ChallengeActionType.ALLOW;
+import static com.thomashan.coup.action.ChallengeActionType.CHALLENGE;
+import static com.thomashan.coup.action.MainActionType.ASSASSINATE;
+import static com.thomashan.coup.action.MainActionType.EXCHANGE;
+import static com.thomashan.coup.card.Card.AMBASSADOR;
+import static com.thomashan.coup.card.Card.ASSASSIN;
+import static com.thomashan.coup.card.Card.CAPTAIN;
+import static com.thomashan.coup.card.Card.CONTESSA;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class StandardGameScenarioTest {
+    @Test
+    public void test_AssassinateChallenger_Challenge_ChallengeSuccess() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unit tests should not contain more than 1 assert(s).](https://app.codacy.com/app/thomashan/coup-java/pullRequest?prid=4561385)